compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirDiagnosticRenderers.kt

```diff
@@ -105,4 +108,6 @@ object FirDiagnosticRenderers {
             "$list $branches or 'else' branch instead"
         }
     }
+
+    val NOT_RENDERED = Renderer<Any?> { "" }
```

Comment:
I feel like I saw this before...? or maybe some other PR which isn't merged?

Reply:
Yep, it's in #4338 and it's not merged yet.

Reply:
oh... so my memory is half correct. 🤦